plugins/orchestrator/src/components/WorkflowInstancePageContent.tsx

```diff
@@ -50,6 +50,8 @@ const useStyles = makeStyles(_ => ({
   },
   middleRowCard: {
     height: '20rem',
+    overflow: 'overlay',
```

Comment:
Reading about it, seemss like "overlay" value is deprecated
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow

Reply:
good to know, changing to "auto" then